guide/16-geoenrichment-guides/part4_enrich_collections_and_sedf.ipynb

```diff
@@ -0,0 +1,2937 @@
+{
```

Comment:
<p>make heading one level less to nest it under previous heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
<p>Leaving this as is for now. The idea is to show a couple of examples of enriching using data collections (one subsection) and analysis variables (other subsection). I don't think nesting under these sub sections is necessary but if you feel otherwise, I can make the change.</p>